### PR TITLE
added doc for Kandinsky3.0

### LINKED ISSUE #5936: [doc] add a doc page for kandinsky 3.0!
we recently added kandinsky 3.0 to diffusers https://github.com/huggingface/diffusers/pull/5913

one of the to-do remaining is to add a doc page for kandinsky 3.0. 
Anyone up to it?

 
current dog page: https://huggingface.co/docs/diffusers/main/en/api/pipelines/kandinsky3#kandinsky-3

info about the model:
github:https://github.com/ai-forever/Kandinsky-3
model weights:https://huggingface.co/kandinsky-community/kandinsky-3
blogpost: https://habr.com/ru/companies/sberbank/articles/775590/

### PR DESCRIPTION
**Pull request by charchit7**
# What does this PR do?

Fixes #5936

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@patrickvonplaten @yiyixuxu

### CODE REVIEW

File: docs/source/en/api/pipelines/kandinsky3.md [hunk 1]
```diff
@@ -9,7 +9,25 @@ specific language governing permissions and limitations under the License.
 
 # Kandinsky 3
 
-TODO
+Kandinsky 3 is created by [Arkhipkin Vladimir](https://github.com/oriBetelgeuse), [Igor Pavlov](https://github.com/boomb0om), [Andrei Filatov](https://github.com/anvilarth), [Zein Shaheen](https://github.com/zeinsh).
+
+The description from it's Github page: 
+
+*Kandinsky 3.0 is an open-source text-to-image diffusion model built upon the Kandinsky2-x model family. In comparison to its predecessors, Kandinsky 3.0 incorporates more data and specifically related to Russian culture, which allows to generate pictures related to Russin culture. Furthermore, enhancements have been made to the text understanding and visual quality of the model, achieved by increasing the size of the text encoder and Diffusion U-Net models, respectively.*
```
Comment (a-r-r-o-w) [inline docs/source/en/api/pipelines/kandinsky3.md:16]:
```suggestion
*Kandinsky 3.0 is an open-source text-to-image diffusion model built upon the Kandinsky2-x model family. In comparison to its predecessors, Kandinsky 3.0 incorporates more data and specifically related to Russian culture, which allows to generate pictures related to Russian culture. Furthermore, enhancements have been made to the text understanding and visual quality of the model, achieved by increasing the size of the text encoder and Diffusion U-Net models, respectively.*
```

**Review by a-r-r-o-w — APPROVED** [review]

**Comment by patrickvonplaten** [comment]
cc @yiyixuxu

File: docs/source/en/api/pipelines/kandinsky3.md [hunk 2]
```diff
@@ -9,7 +9,25 @@ specific language governing permissions and limitations under the License.
 
 # Kandinsky 3
 
-TODO
+Kandinsky 3 is created by [Arkhipkin Vladimir](https://github.com/oriBetelgeuse), [Igor Pavlov](https://github.com/boomb0om), [Andrei Filatov](https://github.com/anvilarth), [Zein Shaheen](https://github.com/zeinsh).
```
Comment (yiyixuxu) [inline docs/source/en/api/pipelines/kandinsky3.md:12]: Let's add the entire list of authors here https://github.com/ai-forever/Kandinsky-3#authors

File: docs/source/en/api/pipelines/kandinsky3.md (hunk 1 above)
Comment (yiyixuxu) [inline docs/source/en/api/pipelines/kandinsky3.md:16]: Let's instead talk a little bit more about the architecture here. I think mostly:
1. a very power text encoder: [FLAN-UL2](https://huggingface.co/google/flan-ul2)
2. special Unet blocks that's twice deeper but remain same parameter counts
3. movq decoder (same as kandinsky 2)

Comment (yiyixuxu) [inline docs/source/en/api/pipelines/kandinsky3.md:16]:
```suggestion
*Kandinsky 3.0 is an open-source text-to-image diffusion model built upon the Kandinsky2-x model family. In comparison to its predecessors, enhancements have been made to the text understanding and visual quality of the model, achieved by increasing the size of the text encoder and Diffusion U-Net models, respectively.*
```

**Review by yiyixuxu — COMMENTED** [review]
left one comment. Thanks for adding this!

File: docs/source/en/api/pipelines/kandinsky3.md (hunk 2 above)
Comment (charchit7) [reply docs/source/en/api/pipelines/kandinsky3.md:12]: Ahh, I missed it. @yiyixuxu Thank you for pointing out :)

File: docs/source/en/api/pipelines/kandinsky3.md [hunk 3]
```diff
@@ -9,7 +9,32 @@ specific language governing permissions and limitations under the License.
 
 # Kandinsky 3
 
-TODO
+Kandinsky 3 is created by [Vladimir Arkhipkin](https://github.com/oriBetelgeuse),[Anastasia Maltseva](https://github.com/NastyaMittseva),[Igor Pavlov](https://github.com/boomb0om),[Andrei Filatov](https://github.com/anvilarth),[Arseniy Shakhmatov](https://github.com/cene555),[Andrey Kuznetsov](https://github.com/kuznetsoffandrey),[Denis Dimitrov](https://github.com/denndimitrov), [Zein Shaheen](https://github.com/zeinsh)[Julia Agafonova]().
```
Comment (patrickvonplaten) [inline docs/source/en/api/pipelines/kandinsky3.md:12]:
```suggestion
Kandinsky 3 is created by [Vladimir Arkhipkin](https://github.com/oriBetelgeuse),[Anastasia Maltseva](https://github.com/NastyaMittseva),[Igor Pavlov](https://github.com/boomb0om),[Andrei Filatov](https://github.com/anvilarth),[Arseniy Shakhmatov](https://github.com/cene555),[Andrey Kuznetsov](https://github.com/kuznetsoffandrey),[Denis Dimitrov](https://github.com/denndimitrov), [Zein Shaheen](https://github.com/zeinsh)
```

Comment (charchit7) [reply docs/source/en/api/pipelines/kandinsky3.md]: Hey @patrickvonplaten Julia Agafonova is mentioned in the author's list (https://github.com/ai-forever/Kandinsky-3#authors) but her github account is not there.

**Comment by patrickvonplaten** [comment]
Great job @charchit7

**Comment by charchit7** [comment]
Thanks @patrickvonplaten First PR merged to diffusers! more to go.

**Pull request merged**